book/website/collaboration/research-infrastructure-roles/community-manager.md

```diff
@@ -15,42 +16,60 @@ _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licen
+They are often involved in culture change to support collaboration between members of the community, through the development of community leadership and agency. 
+They value the people doing the research, and develop structures to support the generation of bottom-up activity <mark>does bottom-up need a definition/glossary listing?</mark>. Their goals are to:
 
-Each role is varied but the main activities are typically focused around compassion, stewardship, and collaboration with the community: 
-* Catalyse connections with all stakeholders
-* Foster communities (integrating communities together)
-* Linking the right people up together (based on expertise and interests)
-* Active community member
-* Communicating technical topics to non-technical colleagues and vice versa
+* Embed open, inclusive and reproducible research practices
+* Ensure a shared understanding of goals, roadmap, processes
+* Facilitate stakeholder engagement and collaboration
+* Provide technical support and domain expertise
+* Co-create, maintain and communicate project resources
+* Amplify and champion community learnings and achievements
+
+Each role is varied but the main activities are typically focused around advocacy for the community, stewardship, scaffolding for collaboration, and the flow of information across all levels of the research, both to and from leadership. and collaboration with the community. 
+
+Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal Code of Conduct <mark>link to glossary?</mark>. 
+The aim of this work is to understand and gather learnings about how and where teams can work well together and diagnose issues which have contributed to challenges in collaboration. The Research Community Manger should then be well positioned to develop systems to support effective and inclusive collaboration, and mitigate harm to individuals or groups of the community. 
+
+
+Themes of Research Community Manager work can be:
+* Catalysing connections with all stakeholders
+* Fostering relationships within and between communities (integrating communities together)
+* Linking the right people up together (based on expertise, interests and project goals)
+* Identifying and growing engaged community members into community leaders
+* Bridging between technical and non-technical members of the community by building and delivering training
 * Encouraging high standards of reproducible, ethical, inclusive and collaborative data science
-* Creating sub-networks within the community around shared experiences, for example, such as an early career researcher network
+* Facilitating the creation of sub-networks within the community around shared experiences, for example, such as an early career researcher network
```

Comment:
(remove 'such as')
* Facilitating the creation of sub-networks within the community around shared experiences, for example, an early career researcher network